test/isochrone.cc

```diff
@@ -370,15 +370,15 @@ TEST(Isochrones, test_geotiff_output_distance) {
   std::string geotiff = thor_worker.isochrones(request_pbf);
 
   std::string name = "/vsimem/test_isogrid_geotiff_d.tif";
-  unsigned char buffer[geotiff.length()];
```

Comment:
This is UB I think.

MSVC barked at this expecting constant expression, which makes sense. This is dynamic allocation.